policyengine_canada/variables/gov/cra/benefits/old_age_security_pension/old_age_security_pension_repayment.py

```diff
@@ -0,0 +1,25 @@
+from policyengine_canada.model_api import *
+
+# SPSD/M 29.0: imioas or imoasmax, I'm not sure. I think imioas is post-repayment?
+
+
+class old_age_security_pension_repayment(Variable):
+    value_type = float
+    entity = Person
+    label = "Old age security pension repayment"
+    documentation = "Your repayment calculation is based on the difference between your income and the threshold amount for the year. The first step is to figure out how much higher your income is than the threshold. You must repay 15 percent of that amount. See OASTD and OASRR in the SPSD/M Parameter Guide"
+    unit = CAD
+    definition_period = YEAR
+
+    def formula(person, period, parameters):
+        p = parameters(period).gov.cra.benefits.old_age_security_pension.amount
```

Comment:
I'd suggest this to go with the scale parameter above:
```suggestion
        p = parameters(period).gov.cra.benefits.old_age_security_pension
        # The income concept in the documentation is 'net world income'. I just use net income here.
        net_income = person("individual_net_income", period)
        # The repayment tax is a percent of income above a threshold.
        uncapped_repayment_tax = p.repayment_tax.calc(net_income)
        # Cap repayment at the pre-repayment amount.
        oas_pre_repayment = person(
            "old_age_security_pension_pre_repayment", period
        )
        repayment_tax = min_(oas_pre_repayment, repayment_tax_uncapped)
        return numpy.around(repayment_tax, 2)
```
Note that we use `min_`, which is short for `numpy.minimum`. `min` won't work as it's not vectorized.

Reply:
See my comment above for why I think this doesn't quite work -- the repayment is based on a percentage of the difference between net income and the threshold, not a percentage of net income.